### PR TITLE
Check for PR merges every 5 minutes

### PR DESCRIPTION
**Pull request by TooAngel**

### CODE REVIEW

**Comment by democratic-collaboration** [comment]
[democratic collaboration](https://github.com/TooAngel/democratic-collaboration)
Approved reviews will speed up the merge, request changes will slow it down.

Please review the PR to help.

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 65/68
    Coefficient: 0.955882352941
    Merging in 0 days 9 hours
    Age 0 days 0 hours

File: src/server.py [hunk 1]
```diff
@@ -291,7 +293,7 @@ def check_pull_request(repository, pull_request, commentOnIssue):
 
 
 def check_pull_requests():
-    print(datetime.now())
+    print('Check pull requests:', datetime.now())
```
Comment (TooAngel) [inline src/server.py]: k, thx

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 68/69
    Coefficient: 0.985507246377
    Merging in 0 days 2 hours
    Age 0 days 1 hours

**Pull request merged**